lib/python/milc.py

```diff
@@ -308,7 +308,7 @@ def description(self, value):
         self._description = self._arg_parser.description = value
 
     def echo(self, text, *args, **kwargs):
-        """Print colorized text to stdout.
+        """Print text to stdout.
 
         ANSI color strings (such as {fg-blue}) will be converted into ANSI
```

Comment:
To match the regex used
```suggestion
        ANSI color strings (such as {fg_blue}) will be converted into ANSI
```